packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json

```
@@ -12,6 +12,8 @@
   "ip_ipv4": "IPv4",
   "ip_ipv6": "IPv6",
   "ip_ipblock": "IP Block",
+  "ipv4_filter": "Afficher Ies IPV4",
+  "ipv6_filter": "Afficher Ies IPV6",
```

```suggestion
  "ipv4_filter": "Afficher les IPV4",
  "ipv6_filter": "Afficher les IPV6",
```
it isn't upper case I (i) but lowercase l (L)

---

Done